documentation/src/main/markdown/currentreleasenotes.md

```diff
@@ -6,3 +6,6 @@
 * ReversingLabs Scans - this new feature provides analysis of software packages for file-based malware threats.
 	See [ReversingLabs Scans](runningdetect/threatintelscan.md) for further information.
 
+## Dependency updates
+* Upgraded Component Locator to version 1.1.10 to certify support for location of components in Yarn Lock and Nuget Centralized Package Management files.
```

Comment:
In the past release notes, this was referred to as " Component Location Analysis". And mentions of added support was in the New Features section.

Reply:
This is an attempt to bring consistency based on how we provide updates in release notes to Detect features versus features of a dependency.
The name of the library is 'component-locator' in the build.gradle file. Would it not help to use the actual name to cite version change? We could also refer to both the names.

Reply:
I think we should restrict the "Dependency updates" section to true 3rd party dependency upgrades. To our end users, component-locator library is an internal implementation. It would make sense to me that we would put something like: _"Component Location Analysis upgraded to certify support for location of components in Yarn Lock and Nuget Centralized Package Management files."_ in the New Features section.

Reply:
Done, thanks!